### DISCUSSION TITLE
V0.13 docker image? - "stable" is v012, "latest" returns "Error response from daemon: manifest unknown"

### DISCUSSION THREAD

**Discussion by douginoz** [blakeblackshear/frigate#9761] [Ask A Question]
I'm trying to update my docker compose file for v013.  If I use the existing
"image: ghcr.io/blakeblackshear/frigate:stable", it is Frigate v0.12.  If I use ":latest" get the manifest unknown error.

What's the correct image: value to use for v0.13?

**Comment by blakeblackshear** [comment]
The stable tag is 0.13. You probably just have a cached image. You will need to run the following to pull updated images:
```
docker compose pull
docker compose up -d
```

This is how it works for any docker image you use for any other applications as well.
